scripts/run_stubtest.py

```diff
@@ -85,9 +85,11 @@
     ]
 
     # create allowlist
-    with tempfile.NamedTemporaryFile(mode="w+t") as allow:
-        allow.write("\n".join(_ALLOWLIST))
-        allow.flush()
+    with tempfile.TemporaryDirectory() as td:
+        allow = os.path.join(td, "test")
+        with open(allow, "w+t") as allow:
+            allow.write("\n".join(_ALLOWLIST))
+            allow.flush()
```

Comment:
On Windows, creating a temporary file and leaving it open means it can't be opened by another function.  The above solves that problem.